client/me/purchases/list/site/index.jsx

```diff
@@ -14,6 +14,7 @@ import SectionHeader from 'components/section-header';
 
 const PurchasesSite = ( { isPlaceholder, name, purchases, slug } ) => {
 	let items, label = name;
+	const siteUrl = slug && slug.replace( /::/g, '/' );
```

Comment:
`site.domain` only contains the host name, not the path: `www.scruffian.com`. The full object looks like this:
```
Object { domain: "www.scruffian.com", id: 83383441, name: "Jetpack Test", slug: "www.scruffian.com::jetpack-test", title: "Jetpack Test", purchases: Array[1] }
```